README.md

```diff
@@ -133,6 +133,21 @@ set -e
 
 On `pre-commit` git phase, the hook triggers the `git-code-format:on-pre-commit` which formats the code of the modified java files using `google-java-format`.
 
+### Configuring Maven Executable Path
+By default, the plugin will generate the following maven path
+
+```shell
+"${env.M2_HOME}/bin/mvn"
+```
```

Comment:
As you can see in MavenEnvironment, this is not true :)

Reply:
Yes, sorry.
`MavenEnvironment` indicates use of `maven.home`, 
whereas README says `${env.M2_HOME}/bin/mvn` in **How the hook works** section. 
I'll remove this line, but maybe the other sections needs updating too?